### DISCUSSION TITLE
group auto-zoom behavior with "primary" node setting camera position?

### DISCUSSION THREAD

**Discussion by hatmix** [ramokz/phantom-camera#313] [Question / Help Wanted]
Is it possible to get the group follow auto-zoom behavior while having camera position determined by one of the nodes in the follow group (like a "primary", e.g. the player)?

**Comment by ramokz** [comment]
At the minute, the camera's position is determined by the centre point between all the nodes in a `follow group`. Likewise, the `auto zoom` functionality scales to contain the two nodes equally within its viewport.

I guess one option would be to have a weight system like in Cinemachine, where you can tell the camera to lean more towards one particular `follow group` target than others. Essentially offset the camera towards a particular target. Have considered looking into this at some point.
